pkg/vz/errors_darwin.go

```diff
@@ -7,5 +7,5 @@ package vz
 
 import "errors"
 
-//nolint:revive // error-strings
+//nolint:revive,staticcheck // backward compatibility
```

Comment:
Is there anything that greps "Rosetta is ..." ?

Reply:
No, it's not. Created #3335 to fix this.

Reply:
Changed `//nolint` message: it's a false positive (see https://github.com/mgechev/revive/issues/1264).